File: Python/7. 기본 수학 1단계/2292.py
# 1 1*6 2*6 3*6 4*6 5*6
# 2~7
# 8~19
# 20~37
a=int(input())
sum=1
cnt=1

while(True):
    if(sum<a):
        sum+=cnt*6
        cnt+=1
    else:
        break

print(cnt)

File: C++/7. 기본 수학 1단계/2292.cpp
#include <iostream>

using namespace std;

int main(){
    int n, i=1, cnt=1, ans=1;
    scanf("%d", &n);

    while(1){
        if(cnt>=n)
            break;
        cnt+=i*6, i++, ans++;
    }

    printf("%d", ans);

    return 0;
}